test_util/cluster_api.py

```diff
@@ -338,6 +340,17 @@ def get_test_app_in_docker(self, ip_per_container):
         return app, test_uuid
 
     def deploy_test_app_and_check(self, app, test_uuid):
+        """This method deploys the test server app and then
+        pings its /operating_environment endpoint to retrieve the container
+        user.
+
+        In a docker container this will be whatever is set as USER (defaults to root)
```

Comment:
nit: This comment suggests the code should be looking up USER from the docker but it doesn't. Maybe something like "Docker containers run things as USER from the Dockerfile, That is currently in these tests always the default, root"".